src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -535,4 +542,110 @@
      */
     Collection<BannerPatternShape> getBannerPatternShapes();
 
+    /**
+     * Gets the {@link TextColor} with the provided name.
+     *
+     * @param name The name of the text color
+     * @return The {@link TextColor} with the given name or Optional.absent() if not found
```

Comment:
Changing this for all methods in the GameRegistry (consistency) is really out of scope for this PR